passenger_exporter.go

```diff
@@ -22,19 +21,121 @@ import (
 	"github.com/prometheus/common/version"
 )
 
-const (
-	namespace = "passenger_nginx"
+// Info represents the info section of passenger's status.
+type Info struct {
+	PassengerVersion         string       `xml:"passenger_version"`
+	AppCount                 string       `xml:"group_count"`
+	CurrentProcessCount      string       `xml:"process_count"`
+	MaxProcessCount          string       `xml:"max"`
+	CapacityUsed             string       `xml:"capacity_used"`
+	TopLevelRequestQueueSize string       `xml:"get_wait_list_size"`
```

Comment:
Catching up a little: I'm assuming we use `get_wait_list_size` as possible the main metric for scaling?

Reply:
Yes! However, we're not using _that_ `get_wait_list_size`. Instead we're using [this one](https://github.com/Intellection/passenger-exporter/pull/6/files#diff-283d9b8a6d990144fb962387b831ac11R56).

Reply:
@tsu-shiuan That is, the one in the `Group` struct, not the `SuperGroup`. Read [this][1].

[1]: https://www.phusionpassenger.com/library/admin/standalone/overall_status_report.html#viewing-process-and-request-queue-information